src/ipc/candidates/edge_edge.cpp

```diff
@@ -60,6 +60,7 @@ bool EdgeEdgeCandidate::ccd(
     const Eigen::MatrixXi& E,
     const Eigen::MatrixXi& F,
     double& toi,
+    const double min_distance,
```

Comment:
doesn't look like you are using this parameter

Reply:
Should be added to call below, fixed.